src/fr/charly/javaSwingTutorial/JavaSwingLesson2BorderLayout.java

```diff
@@ -0,0 +1,50 @@
+package fr.charly.javaSwingTutorial;
+
+import java.awt.BorderLayout;
+import java.awt.Dimension;
+
+import javax.swing.JButton;
+import javax.swing.JCheckBox;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+import javax.swing.JTextArea;
+import javax.swing.UIManager;
+import javax.swing.plaf.nimbus.NimbusLookAndFeel;
+
+public class JavaSwingLesson2BorderLayout extends JFrame {
+
+	private static final long serialVersionUID = 7768498545170471759L;
+
+	public JavaSwingLesson2BorderLayout() {
+		super("My first Swing application");
+		this.setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE);
+		this.setSize(600, 400);
+		this.setLocationRelativeTo(null);
+
+		JPanel contentPane = (JPanel) this.getContentPane();
+		// contentPane.setLayout(new BorderLayout();
```

Comment:
Why is this code commented? If it is not needed, remove it. If it is needed, then remove the comments.